src/routes/get-started/developers/upgrade-guide/AngularGuide.tsx

```diff
@@ -0,0 +1,413 @@
+import { GoACallout, GoATable } from "@abgov/react-components";
+import { CodeSnippet } from "@components/code-snippet/CodeSnippet.tsx";
+import { InlineCode } from "@components/inline-code/InlineCode.tsx";
+import { components } from "@routes/get-started/developers/upgrade-guide/components.ts";
+
+export const AngularGuide = () => {
+  const angularComponents = components.filter(x => x.angular);
+  const componentsWithTemplateRef = angularComponents.filter(component =>
+    component.angular?.some(prop => prop.v4?.type.includes("TemplateRef"))
+  );
+  return (
+    <>
+      <h2 id="angular">Migrating an Angular app</h2>
+      <GoACallout>
+        <p>
+          <InlineCode>@abgov/angular-components</InlineCode> supports only Angular v16 and above.
+        </p>
+      </GoACallout>
+
+      <h4>1. Update dependencies</h4>
+      <CodeSnippet
+        lang="typescript"
+        allowCopy={true}
+        code={`
+          npm i @abgov/web-components@latest
+          npm i @abgov/angular-components@latest
+          npm i @abgov/ui-components-common@latest
```

Comment:
The `ui-components-common` has been moved into the React lib, so it is no longer needed here.